recipe/meta.yaml

```diff
@@ -7,8 +7,8 @@ source:
     git_tag: ESMF_7_0_0
 
 build:
-    number: 1
-    skip: True  # [win or py3k]
+    number: 2
+    skip: True  # [win]
```

Comment:
@bekozi you will need to re-render to get Python 3 in the build matrix. See http://conda-forge.github.io/docs/conda_smithy.html